lib/SimplePie/SimplePie/Parser.php

```diff
@@ -181,12 +181,20 @@ public function parse(&$data, $encoding, $url = '')
 			xml_set_element_handler($xml, 'tag_open', 'tag_close');
 
 			// Parse!
-			if (!xml_parse($xml, $data, true))
-			{
-				$this->error_code = xml_get_error_code($xml);
-				$this->error_string = xml_error_string($this->error_code);
-				$return = false;
-			}
+			$stream = fopen('php://memory','r+');                             
```

Comment:
@Kiblyn11 Maybe we should consider `php://temp` instead. What do you think? https://php.net/wrappers.php

Reply:
@Alkarex yeah sounds better than putting that into memory, have to make sure it's cleaned properly though.
Will look into it.